src/main/java/core/basesyntax/db/FruitShopDao.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.db;
+
+import java.util.List;
+
+public interface FruitShopDao {
+    int storageSize();
+
+    List<String> getKeyAndValue();
+
+    void put(String fruit, int quantity);
+
+    void putOnlyFruits(String fruit);
+}
```

Comment:
The interface methods should have Javadoc comments to explain their purpose and usage, which is especially important for public interfaces as part of an API.